config/kubernetes-sigs/sig-cluster-lifecycle/teams.yaml

```diff
@@ -341,20 +341,25 @@ teams:
   cluster-api-release-team:
     description: Release team of Cluster API
     members:
-    - aniruddha2000
+    # members added in commented lines have a pending membership
+    # and will be added back once it is acquired.
+    # - adilGhaffarDev
+    - cahillsf
     - CecileRobertMichon
```

Comment:
thoughts on removing @sbueringer and I now that we have an official maintainer role?

Reply:
Sounds good to me

Reply:
>thoughts on removing @sbueringer and I now that we have an official maintainer role?

done